dbt_meshify/dbt_projects.py

```diff
@@ -152,7 +159,19 @@ def get_catalog_entry(self, unique_id: str) -> Optional[CatalogTable]:
 
     def get_manifest_node(self, unique_id: str) -> Optional[ManifestNode]:
         """Returns the catalog entry for a model in the dbt project's catalog"""
-        return self.manifest.nodes.get(unique_id)
+        if unique_id.split(".")[0] in [
```

Comment:
What about `exposure` and `metric`/`measure`? Are those not listed on purpose?

Reply:
this logic was to get `nodes` vs the resources that are also other top-level keys in the manifest. Looking with fresh eyes, I would guess there's a dbt-core class that represents this type that we could leverage instead of a list of strings!